modules/core/client/css/lightbox.css

```diff
@@ -26,3 +26,17 @@
     left:auto;
     right: 0;
 }
+
+.lightbox-app {
+  position: fixed;
+  /*max-height: 500;*/
+  top: 5%;
+  left: 10%;
+  right: 10%;
+  padding: 10px;
+  background-color: white;
+  border: 2px solid black;
+  z-index: 2;
+  overflow-x: hidden;
+  overflow-y: scroll;
+}
```

Comment:
Rather than re-applying all of this, you can add another class to the element in addition to lightbox and just overwrite the max-height value